src/indexer/utils/substrate/index.ts

```diff
@@ -124,10 +114,9 @@ export async function saveDeposit(
   } = substrateDepositData
 
   const decodedAmount = getDecodedAmount(depositData)
-
-  const foundTransfer = await transferRepository.findByNonceFromDomainId(Number(depositNonce), `${originDomainId}`)
-
-  if (foundTransfer) {
+  let transfer: Transfer | null
```

Comment:
seems redundant considering the following line

Reply:
The following line can return null